syntax = "proto3";


service WebcamService {


    rpc CaptureImage (ImageRequest) returns (ImageResponse){}
    
    rpc ImageStream (stream ImageRequest) returns (stream ImageResponse){}
}



message ImageRequest {
    string device_id = 1;
}


message ImageResponse {
    
    bytes image_data = 4;
}

